config.toml

```diff
@@ -55,9 +55,14 @@ url = "https://github.com/ipfs/go-ipfs/blob/master/docs/file-transfer.md"
 weight = 2
 [[menu.guides]]
 parent = "guides"
+name = "Setting Up an IPFS Gateway"
```

Comment:
```suggestion
name = "Setting Up a Gateway"
```

Reply:
What if we shortened it to this, and relied on the context of the page to communicate the IPFS-ness?